tubular/scripts/frontend_utils.py

```diff
@@ -127,6 +153,20 @@ def create_version_file(self):
         except IOError:
             self.FAIL(1, 'Could not write to version file for app {}.'.format(self.app_name))
 
+    def copy_js_config_file_to_app_root(self, app_config, app_name):
+        """ Creates a copy of env.config.js(x) file to the app root """
+        source = app_config.get('JS_CONFIG_FILEPATH')
+        filename = app_config.get('LOCAL_CONFIG_FILENAME', "env.config.js")
```

Comment:
[nit/suggestion] It might be worth renaming `LOCAL_CONFIG_FILENAME` to `LOCAL_JS_CONFIG_FILENAME` so the config setting is explicitly referencing the JS config (versus some other config).

Reply:
Done.